set.mm

```diff
@@ -677842,7 +677842,7 @@ collection and union ( ~ mnuop3d ), from which closure under pairing
           -> -. A. i ( i e. z -> ( v e. U -> ( i e. v -> ( v e. f ->
             -. A. u ( u e. f -> ( i e. u ->
               -. A. o ( o e. u -> A. s ( s e. o -> s e. w ) ) ) ) ) ) ) ) ) ) )
-    ) $=
+      ) $=
```

Comment:
Can you merge the last two lines of the statement, i.e.,
```
            -. A. o ( o e. u -> A. s ( s e. o -> s e. w ) ) ) ) ) ) ) ) ) ) ) ) $=
```

Reply:
(same with statements below)

Reply:
I changed some of them, but for the others, doing that would exceed the 79-characters-per-line limit.